tests/check_consistent.py

```diff
@@ -21,6 +21,7 @@
     {"stdlib/@python2/builtins.pyi", "stdlib/@python2/__builtin__.pyi"},
     {"stdlib/threading.pyi", "stdlib/_dummy_threading.pyi"},
 ]
+metadata_keys = {"version", "python2", "python3", "requires", "extra_description", "obsolete_since"}
```

Comment:
What's `extra_description`? It doesn't seem to be used anywhere else in this PR.

Reply:
It's unused at the moment, but I added it in typeshed-internal/stub_uploader#11 for future use.

Reply:
I plan to document `METADATA.toml`, but I need to resolve #5422 first.